coops-ui/src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <v-card flat class="container">
+    <div class="meta-container">
+      <label>
+        <span>Director Change Date</span>
+      </label>
+
+      <div class="value date">
+        <v-menu
+          ref="menu"
+          v-model="menu"
+          :nudge-right="40"
+          transition="scale-transition"
+          offset-y
+          min-width="18rem">
+          <template v-slot:activator="{ on }">
+            <v-text-field
+              id="cod-textfield"
+              v-model="dateFormatted"
+              :rules="codDateRules"
+              label="Date"
+              hint="YYYY/MM/DD"
+              append-icon="mdi-calendar"
+              v-on="on"
+              filled data-test-id="cod-date-text">
+            </v-text-field>
+          </template>
+          <v-date-picker
+            id="cod-datepicker"
+            v-model="date"
+            :min=minDate
+            :max=maxDate
+            no-title
+            @input="menu = true" data-test-id="cod-date-picker">
```

Comment:
Thanks for adding test id. Could you please move this attribute to its own line near the id above?

Reply:
Resolved in PR #556.